src/ServiceProviders/SiteProviders/PantheonProvider.php

```diff
@@ -98,6 +98,9 @@ public function credentialRequests()
         $adminEmailRequest = (new CredentialRequest('ADMIN_EMAIL'))
             ->setRequired(false);
 
+        $adminEmailRequest = (new CredentialRequest('ADMIN_USERNAME'))
```

Comment:
Copy/paste issue? I'm assuming this should be `$adminUsernameRequest`

Reply:
Whoops yes this was a copy/paste error. I've corrected the issue and performed some manual test to ensure it didn't introduce any issues both with the username specified and without.